src/shared/components/person/person-details.tsx

```diff
@@ -107,16 +107,6 @@ export class PersonDetails extends Component<PersonDetailsProps, any> {
     setupTippy();
   }
 
-  // TODO wut?
```

Comment:
This was 3 years old, commented out, and I have not idea what to make of "TODO wut?" so I removed it.